app/src/main/java/org/mozilla/focus/utils/ViewUtils.kt

```diff
@@ -95,12 +95,16 @@ object ViewUtils {
         val snackbar = Snackbar.make(view, resId, Snackbar.LENGTH_LONG)
 
         val snackbarView = snackbar.view
-        snackbarView.setBackgroundColor(ContextCompat.getColor(context, R.color.snackbarBackground))
-
         val snackbarTextView = snackbarView.findViewById<View>(R.id.snackbar_text) as TextView
         snackbarTextView.setTextColor(ContextCompat.getColor(context, R.color.snackbarTextColor))
-        snackbarTextView.gravity = Gravity.CENTER
-        snackbarTextView.textAlignment = View.TEXT_ALIGNMENT_CENTER
+        if (MvpFeatureManager.isEnabled) {
+            snackbarView.setBackgroundResource(R.drawable.background_snackbar)
+            snackbarTextView.gravity = Gravity.CENTER_VERTICAL
```

Comment:
The alignment and startPadding seem off. CENTER_VERTICAL doesn't seem to have worked but I haven't looked into why just yet.

This PR:
<img width="518" alt="Screen Shot 2021-07-15 at 6 57 42 PM" src="https://user-images.githubusercontent.com/1370580/125868169-68293d8a-30fc-4f18-ac16-a8b5e493cd5d.png">

Designs:
<img width="486" alt="Screen Shot 2021-07-15 at 6 57 58 PM" src="https://user-images.githubusercontent.com/1370580/125868168-744963af-e1f2-4b01-94c7-dfe316796c55.png">